### ISSUE TITLE
Deployment GUI Dashboard

### ISSUE THREAD

**Issue by SHIV5T3R** [sanger/General-Backlog-Items#457]
**User story**
I think that a GUI dashboard dedicated to handling and displaying output for a deployment would be a nice-to-have for the team. This will build upon what @StephenHulme has created with the deployment params. 

There is a [PR](https://github.com/sanger/deployment/pull/493) open for starting a deployment from the new GUI but it's in a bare state right now.

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Starting a deployment with sufficient protection and validation
- [ ] Viewing the deployment logs from within the dashboard
- [ ] Control over the deployment from the GUI (e.g, cancelling a deployment) - (should)
- [ ] Auto-completing release tags - (should)

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/deployment/pull/493
